src/main/java/org/openstreetmap/atlas/geography/MultiPolygon.java

```diff
@@ -99,11 +99,10 @@ public GeoJsonObject asGeoJson()
     public GeoJsonObject asGeoJsonFeatureCollection()
     {
         final GeoJsonBuilder builder = new GeoJsonBuilder();
-        return builder.createFeatureCollection(Iterables.translate(outers(), outerPolygon ->
-        {
```

Comment:
oh common sonar, have mercy, I did this for clarity

Reply:
Sonar has spoken. Question not its wisdom. :D